задание4

s-buk opened the pull request.

s-buk committed:
задание4

s-buk committed:
задание4 NEW

s-buk committed:
Доделал Задание4

akartynnik commented on задание4/style.css:
```diff
@@ -0,0 +1,51 @@
+﻿@font-face {
+    font-family: Sasha2;
+    src: url(fonts/pattaya-regular.ttf);
```
не забываем путь оборачивать в кавыки (одинарные или двойные) `src: url('fonts/pattaya-regular.ttf');`

akartynnik commented on задание4/style.css:
```diff
@@ -0,0 +1,51 @@
+﻿@font-face {
+    font-family: Sasha2;
```
не забываем имя нового шрифта оборачивать в кавыки (одинарные или двойные) `font-family: "Sasha2"`

akartynnik commented on задание4/style.css:
```diff
@@ -0,0 +1,51 @@
+    border: 1px solid rgb(0, 0, 0);
+    font-style: italic;
+}
+table {border-collapse: collapse;}
```
Нет отступов

akartynnik commented on задание4/style.css:
```diff
@@ -0,0 +1,51 @@
+}
+
+table {border: 1px solid rgb(0, 0, 0);} 
+td {border: 1px solid rgb(0, 0, 0);} 
```
Нет отступов

akartynnik commented on задание4/style.css:
```diff
@@ -0,0 +1,51 @@
+    text-transform: uppercase;
+}
+
+table {border: 1px solid rgb(0, 0, 0);} 
```
Нет отступов

akartynnik commented on задание4/style.css:
```diff
@@ -0,0 +1,51 @@
+
+img {
+     height:200px;
+  }
```
лишние пробелы перед скобкой

akartynnik commented on задание4/style.css:
```diff
@@ -0,0 +1,51 @@
+th {
+    border: 1px solid rgb(0, 0, 0);
+    font-style: italic;
+}
```
после закрывающей скобки ставим пробельную строку

s-buk committed:
убрал лишние пробелы, добавил пустые строки и кавычки.

s-buk committed:
Добавил новый файл "задание4/images/background-image.jpg". Описание
планет обернул в div-блоки с центровкой содержимого. Заливка фона в блоках. Изменил цвет текста в блоке и заголовках. В шапке таблицы изменил заливку и цвет текста. Изменил расстояние между границей ячейки и ее содержимым в таблице (добавил атрибут "cellpadding" для таблицы).

akartynnik commented on задание4/index.html:
```diff
@@ -0,0 +1,124 @@
+        </p>
+
+    </div>
+    </ul>
```
откуда взялся ul закрывающий?

akartynnik commented on задание4/index.html:
```diff
@@ -0,0 +1,124 @@
+
+    </div>
+
+    <div style="color: rgb(255, 255, 255); background-image: url(images/background-image.jpg); background-size: cover">
```
фон должен быть фиксированным по заданию

s-buk committed:
добавил свойство background-attachment для фиксации фона

s-buk committed:
задание7
